### PR TITLE
Changes for PCLM, Secret store, developer mode and screenshots changed

### PR DESCRIPTION
**Pull request by Nidhi251289**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 1]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
 Storage plans are “blueprints” that specify how to request/decomission a new database or blob storage and pass its credentials to an environment.
 {{% /alert %}}
 
+The following **Storage Types** are supported:
+
+* Minio
+* Ephemeral
+* Amazon-S3
+* Azure Blob
+* Google-cloud
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Google Cloud Storage bucket

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 2]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
 Storage plans are “blueprints” that specify how to request/decomission a new database or blob storage and pass its credentials to an environment.
 {{% /alert %}}
 
+The following **Storage Types** are supported:
+
+* Minio
+* Ephemeral
+* Amazon-S3
+* Azure Blob
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Azure Blob Storage

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 3]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
 Storage plans are “blueprints” that specify how to request/decomission a new database or blob storage and pass its credentials to an environment.
 {{% /alert %}}
 
+The following **Storage Types** are supported:
+
+* Minio
+* Ephemeral
+* Amazon-S3
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Amazon S3

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 4]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
 Storage plans are “blueprints” that specify how to request/decomission a new database or blob storage and pass its credentials to an environment.
 {{% /alert %}}
 
+The following **Storage Types** are supported:
+
+* Minio
+* Ephemeral
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Maybe call this *Ephemeral (non-persistent)*

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 5]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
 Storage plans are “blueprints” that specify how to request/decomission a new database or blob storage and pass its credentials to an environment.
 {{% /alert %}}
 
+The following **Storage Types** are supported:
+
+* Minio
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: MinIO

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 6]
```diff
@@ -348,6 +348,15 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
...
+* Ephemeral
+* Amazon-S3
+* Azure Blob
+* Google-cloud
+* ceph
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Ceph RADOS

**Comment by MarkvanMents** [comment]
@katarzyna-koltun-mx - once this is out of draft, we need to discuss who picks it up and resolves conflicts/proofreads/merges it.
Adding you as assignee for information.

**Comment by MarkvanMents** [comment]
@Nidhi251289 - what is the status of this PR?
Do you know when you are likely to want it reviewed and published?

**Comment by Nidhi251289** [comment]
HI Mark,

This PR contains changes for multiple release such as PCLM, Secret store, some documentation changes. Since, we did not release PCLM yet, i would like to refer this once we release PCLM. After which i will close this PR.

Regards
Nidhi
________________________________

**Comment by MarkvanMents** [comment]
> HI Mark, This PR contains changes for multiple release such as PCLM, Secret store, some documentation changes. Since, we did not release PCLM yet, i would like to refer this once we release PCLM. After which i will close this PR. Regards Nidhi

Thanks for the update. Let me know if you want to split out the PCLM stuff and publish the rest. Otherwise, I'll wait to hear from you.

**Comment by katarzyna-koltun-mx** [comment]
@Nidhi251289 Hey Nidhi, what's the status of this one? Shall I delete it?

**Comment by Nidhi251289** [comment]
@katarzyna-koltun-mx can we delete this PR. This is not requited anymore

**Comment by katarzyna-koltun-mx** [comment]
Closing it should be enough. Thank you!

**Pull request closed without merging**